src/api/common/interceptors/metrics.interceptor.ts

```
@@ -0,0 +1,79 @@
+    );
+  }
+
+  private recordMetrics(
+    context: ExecutionContext,
+    path: string,
+    method: string,
+    start: number,
+    isError: boolean,
+  ): void {
+    const response = context.switchToHttp().getResponse();
+    const status = isError ? 500 : (response.statusCode || 200);
```

ESLint

[error] Replace `(response.statusCode·||·200)` with `response.statusCode·||·200`

(prettier/prettier)